src/mission_end.cpp

```diff
@@ -19,6 +19,15 @@ void mission_end::heal_infection( mission *miss )
     p->remove_effect( effect_infection );
 }
 
+void mission_end::heal_asthmatic( mission *miss )
```

Comment:
It doesn't really do anything, it doesn't need to be added. You can just use the empty mission end here.